frontend/packages/kubevirt-plugin/src/models/index.ts

```diff
@@ -231,3 +231,16 @@ export const HyperConvergedModel = {
   crd: true,
   plural: 'hyperconvergeds',
 };
+
+export const NetworkAddonsConfigModel: K8sKind = {
+  label: 'Network Addons Config',
+  labelPlural: 'Network Addons Configs',
```

Comment:
```suggestion
  label: 'NetworkAddonsConfig',
  labelPlural: 'NetworkAddonsConfigs',
```
I think the new convention is to use the kind also as the label
@glekner do you know if we should use human readable or kind as labels ?

Reply:
yep, use kind as label AFAIK